Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -24,45 +29,32 @@ class PlaceDetailViewController: UIViewController {
                                  maximumActiveDownloads: activeDownloadCount, imageCache: cache)
     }()
 
-    // TODO: make carousel
-    private lazy var imageCarouselLayout: UICollectionViewFlowLayout = {
-        let layout = UICollectionViewFlowLayout()
-        layout.scrollDirection = .horizontal
-        layout.minimumInteritemSpacing = 0
-        layout.minimumLineSpacing = 0
-        return layout
+    fileprivate lazy var panGestureRecognizer: UIPanGestureRecognizer = {
+        let panGesture = UIPanGestureRecognizer(target: self, action: #selector(self.didPan(gestureRecognizer:)))
+        panGesture.maximumNumberOfTouches = 1
+        panGesture.minimumNumberOfTouches = 1
+        return panGesture
     }()
 
-    lazy var imageCarousel: UICollectionView = {
-        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: self.imageCarouselLayout)
-        collectionView.delegate = self
-        collectionView.dataSource = self
-        collectionView.register(ImageCarouselCollectionViewCell.self, forCellWithReuseIdentifier: CellReuseIdentifier)
-        collectionView.backgroundColor = .clear
-        collectionView.showsHorizontalScrollIndicator = false
-        collectionView.showsVerticalScrollIndicator = false
-        collectionView.isPagingEnabled = true
-        return collectionView
-    }()
+    fileprivate var previousCardViewController: PlaceDetailsCardViewController?
+    fileprivate var currentCardViewController: PlaceDetailsCardViewController
+    fileprivate var nextCardViewController: PlaceDetailsCardViewController?
+    fileprivate var unusedPlaceCardViewControllers = [PlaceDetailsCardViewController]()
 
-    lazy var pageControl: UIPageControl = {
-        let pageControl = UIPageControl()
-        pageControl.pageIndicatorTintColor = Colors.pageIndicatorTintColor
-        pageControl.currentPageIndicatorTintColor = Colors.currentPageIndicatorTintColor
-        pageControl.addTarget(self, action: #selector(self.pageControlDidPage(sender:)), for: UIControlEvents.valueChanged)
-        return pageControl
-    }()
+    fileprivate var currentCardViewCenterXConstraint: NSLayoutConstraint?
+    fileprivate var previousCardViewTrailingConstraint: NSLayoutConstraint?
+    fileprivate var nextCardViewLeadingConstraint: NSLayoutConstraint?
```

Comment:
nit: probably not worth changing now because of xcode's inability to refactor but perhaps `trailingCardView...` and `leadingCardView...`?